### PR TITLE
Add Changelog Entry for Teach Lesson

### PR DESCRIPTION
**Pull request by Lyndomen**
## About the PR
Changelog missing from https://github.com/DeltaV-Station/Delta-v/pull/2184

## Why / Balance
It is important to let people know

## Technical details
yaml, but idk if its better of CL it here too

## Media
no

## Requirements

- [X] I have tested all added content and changes.
- [X] I have added media to this PR or it does not require an ingame showcase.

## Breaking changes

**Changelog**
:cl: Lyndomen, SleepyApril
- tweak: Normal kill targets now just need to be killed once- but it needs to teach them a lesson that will be remembered

### CODE REVIEW

File: Resources/Changelog/DeltaVChangelog.yml [hunk 1]
```diff
@@ -3739,3 +3739,10 @@
   id: 683
   time: '2024-11-18T02:08:58.0000000+00:00'
   url: https://github.com/DeltaV-Station/Delta-v/pull/2214
+- author: Lyndomen, SleepyApril
+  changes:
+  - message: Normal kill targets now just need to be killed once- but it needs to teach them a lesson that will be remembered
+    type: Tweak
+  id: 683
```
Comment (deltanedas) [inline Resources/Changelog/DeltaVChangelog.yml]: pro

**Review by deltanedas — CHANGES_REQUESTED** [review]

**Review by MilonPL — CHANGES_REQUESTED** [review]
just add it to this pr's cl and wait for publish

File: Resources/Changelog/DeltaVChangelog.yml (hunk 1 above)
Comment (Lyndomen) [reply Resources/Changelog/DeltaVChangelog.yml]: nothingburger commit

**Comment by Lyndomen** [comment]
if someone can check the CL, i think its the double person that fucked it up fucked if i know

**Review by MilonPL — APPROVED** [review]

**Comment by MilonPL** [comment]
yes real it was the dot in the name

**Comment by Lyndomen** [comment]
> yes real it was the dot in the name

:trollface:

**Review by deltanedas — APPROVED** [review]

**Comment by jimmy12or** [comment]
But... you're not allowed to remember.

**Comment by Lyndomen** [comment]
> But... you're not allowed to remember.

People can tell you "bruh you just got eaten by carp, who did you piss off"

**Comment by Radezolid** [comment]
> But... you're not allowed to remember.

You are not allowed to remember your killer, you can recall being beaten up with a bat "for some weird shit they were yelling that it was my fault" or something alike.

**Pull request merged**